Framework/Algorithms/src/BinaryOperation.cpp

```diff
@@ -492,7 +492,7 @@ void BinaryOperation::doSingleValue() {
 
   if (m_eout) {
     // ---- The output is an EventWorkspace ------
-    PARALLEL_FOR3(m_lhs, m_rhs, m_out)
+    PARALLEL_FOR_IF(Kernel::threadSafe(m_lhs.get(), m_rhs.get(), m_out.get()))
```

Comment:
The single parameter variant now uses a reference. Is there a reason why the 3 parameter version has to call `.get()`?